### ISSUE TITLE
Can't get modal to open

### ISSUE THREAD

**Issue by roybman** [dixso/custombox#175]
I can't figure out what I assume I'm doing wrong.  I can't get a modal to open for me.

HTML box to trigger on click:
<div id="openpostcontent">
<div class="contentbox">
<p>Open Window</p>
</div>
</div>

At the end of the HTML:
<script src="web/js/custombox.min.js"></script>
<script src="web/js/jquery.postcontent.js"></script>
</body>

Contents of web/js/jquery.postcontent.js:
$(function() {
	$('#openpostcontent').on('click', function( e ) {
		Custombox.open({
			target: 'web/postcontent.html',
			overlayClose: false,
			overlaySpeed: 300,
			overlayEffect: corner,
			width: 60,
			effect: 'slide',
			animation: ['top', 'left'],
			speed: 500
		});
		e.preventDefault();
	});
});

postcontent.html is a copy of demo.html.

In the <head> tag I have:
<link rel="stylesheet" href="style.css"> 
<link rel="stylesheet" href="custombox.min.css">
<link rel="stylesheet" href="demo.css">

When I mouse over the "openpostcontent" div, I get a pointer, but get no action on clicking it.
I've gone over this and can't figure why it doesn't work.
Please advise if I'm missing something.
Thanks for any help you can offer.

**Comment by Pikapika555** [comment]
had the same problem
this plugin seems not to be made for the use with 'classical' bootstrap modals
just use anything except a modal as modal ;)

**Comment by roybman** [comment]
Thanks for the input.  I've found since the post that there was a syntax error in the js function (corner needs to be in single apostrophes), but I also encountered other issues once I got passed that.  Primarily, once I got the modal to open, it was opening on the extreme left side of the page, under the elements on my page.  I tried a multitude of adjustments to the options on open, and nothing I changed had any effect on the result.  I found one post elsewhere that explained the documentation on this is not good and it takes a great deal of time to figure out how to use it as a result.  At this point I've given up on this one and moved on.
